webrtc-stats.html

```diff
@@ -268,6 +268,26 @@
           PeerConnection.
         </p>
       </section>
+      <section>
+        <h3>
+          Guidelines for getStats() results caching
+        </h3>
+        <p>
+          A call to getStats() touches many components of WebRTC and may take significant time to
+          execute. An implementation may utilize caching for performance benefits and return the
+          same result as a previous getStats() call.
+        </p>
+        <p>
+          The cache MUST NOT be valid for more than 1 second. The recommended time for cache
+          consistency is no greater than 200 ms.
```

Comment:
The problem we're trying to solve is when something is doing something like
addTrack()
getStats()
And expecting that the thing you changed has shown up.

We should rewrite the guidelines not to say anything about how often the stats update - that is up to the engine - but we should guarantee that objects added/removed are reflected in subsequent getStats calls.

Reply:
For specific events that aren't the direct result of an API call, such as camera dying, then on an observable event like onmuted, subsequent getStats calls need to give updates values for that object.

Reply:
We can add a comment like that for the RTCMediaStreamTrackStats dictionary, and for the guidelines section here say "unless otherwise specified..."